mmv1/products/blockchain/BlockchainNodes.yaml

```diff
@@ -0,0 +1,145 @@
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'BlockchainNodes'
+base_url: projects/{{project}}/locations/{{location}}/blockchainNodes
+self_link: projects/{{project}}/locations/{{location}}/blockchainNodes/{{name}}
+description: |
+  A representation of a blockchain node.
+
+
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/blockchain-node-engine'
+  api: 'https://cloud.google.com/blockchain-node-engine/docs/reference/rest/v1/projects.locations.blockchainNodes'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'firebase_project_basic'
+    min_version: 'beta'
+    primary_resource_id: 'default'
+    vars:
+      instance_name: 'memory-cache'
+      project_name: "my-project"
+    test_env_vars:
+      org_id: :ORG_ID
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  constants: templates/terraform/constants/firebase_project.go.erb
+  pre_create: templates/terraform/pre_create/firebase_project.go.erb
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    output: true
+    description: |
+      The number of the google project that firebase is enabled on.
+  - !ruby/object:Api::Type::String
+    name: createTime
+    output: true
+    description: |
+      The timestamp at which the blockchain node was first created.
+  - !ruby/object:Api::Type::String
+    name: updateTime
+    output: true
+    description: |
+      The timestamp at which the blockchain node was last updated.
+  - !ruby/object:Api::Type::Map
+    name: labels
+    description: |
+      User-provided key-value pairs
+  - !ruby/object:Api::Type::NestedObject
+    name: connectionInfo
+    description: |
+      The connection information through which to interact with a blockchain node.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: endpointInfo
+        output: true
+        description: |
+          The endpoint information through which to interact with a blockchain node.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: jsonRpcApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node JSON-RPC API endpoint.
+          - !ruby/object:Api::Type::String
+            name: websocketsApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node WebSockets API endpoint.
+  - !ruby/object:Api::Type::NestedObject
+    name: ethereumDetails
+    description: |
+      User-provided key-value pairs
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: gethDetails
+        description: |
+          User-provided key-value pairs
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: garbageCollectionMode
+            immutable: true
+            description: |
+              Blockchain garbage collection mode.
```

Comment:
This field is missing a required field called `values` - please take a look at [the contribution guide here](https://googlecloudplatform.github.io/magic-modules/develop/resource/#add-fields) for info about properties needed for the Enum field type (or search the codebase for other examples) and add any missing properties.